### ISSUE TITLE
Ability to change authenticate signingMessage (feature request)

### ISSUE THREAD

**Issue by kkillingbeck** [MoralisWeb3/react-moralis#14]
- **I'm submitting a ...**
  [ ] bug report
  [x] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**

We should be able to change the signing message for authenticate:

`const { user, isAuthenticated, authenticate } = useMoralis();`
`authenticate({signingMessage:"hello"})`

as done with Moralis:

`Moralis.authenticate({signingMessage:"hello"})`

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

**Comment by ivan-liljeqvist** [comment]
Hi use the latest Moralis version as dependency to react and it should work

**Comment by kkillingbeck** [comment]
I'm using moralis@0.0.44 and react-moralis@0.2.2 which is the latest according to npm, is this the right version?

Here is the code I am using:

```
import { Button } from "react-bootstrap";
import { useMoralis } from "react-moralis";

export default function Login(){

	const {isAuthenticated, logout, authenticate} = useMoralis();
	
	return(
		<>
			{!isAuthenticated ? 
				<Button onClick={() => authenticate({signingMessage:"Custom Message"})}>Login</Button> 
			: 
				<Button onClick={logout}>Log Out</Button>
			}
		</>
	)
}
```
signingMessage still reads "Moralis Authentication".

**Comment by ErnoW** [comment]
> I'm using moralis@0.0.44 and react-moralis@0.2.2 which is the latest according to npm, is this the right version?
> 
> Here is the code I am using:
> 
> ```
> import { Button } from "react-bootstrap";
> import { useMoralis } from "react-moralis";
> 
> export default function Login(){
> 
> 	const {isAuthenticated, logout, authenticate} = useMoralis();
> 	
> 	return(
> 		<>
> 			{!isAuthenticated ? 
> 				<Button onClick={() => authenticate({signingMessage:"Custom Message"})}>Login</Button> 
> 			: 
> 				<Button onClick={logout}>Log Out</Button>
> 			}
> 		</>
> 	)
> }
> ```
> 
> signingMessage still reads "Moralis Authentication".

Should be fixed in version 0.2.3

**Comment by kkillingbeck** [comment]
Updated and it works, thank you.
